addons/addons.json

```diff
@@ -149,5 +149,6 @@
   "editor-stage-left",
   "editor-buttons-reverse-order",
   "columns",
-  "editor-theme3"
+  "editor-theme3",
+  "lookatrandom"
```

Comment:
This needs to be above the line that says "NEW ADDONS ABOVE THIS," since this isn't a theme.